### DISCUSSION TITLE
Localized time

### DISCUSSION THREAD

**Discussion by MarcinKlejna** [razonyang/hugo-theme-bootstrap#253] [General]
Hugo 0.87.0 brings time zone support, localized time and dates,  what should i do to make it work with a template?
https://gohugo.io/news/0.87.0-relnotes/

**Comment by razonyang** [comment]
This feature looks good. But I didn't have many experiences about it. I'll give it a shot tonight.

**Comment by razonyang** [comment]
Localization was implemented by latest commit.

After upgrading, you'll need to modify the `dateFormat` parameter, for example:

```
dateFormat = ':date_full'
```

- [Go's layout string](https://gohugo.io/functions/format/#gos-layout-string)
- [Date/time formatting layouts](https://gohugo.io/functions/dateformat/#datetime-formatting-layouts)

> Please keep in mind that the language key should be same as the [locales](https://github.com/gohugoio/locales), otherwise it wont work.

**Comment by razonyang** [comment]
As for the time zone. This theme doesn't use relative functions yet.

Here are some docs useful for you:

- [Time Zone](https://gohugo.io/getting-started/configuration/#timezone)
- [Using Locations](https://gohugo.io/functions/time/#using-locations)
- [Time Zone Database Name](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)

**Comment by MarcinKlejna** [comment]
thank you very much
